lib/httplib/httpheaders.go

```diff
@@ -144,6 +144,13 @@ func SetCacheHeaders(h http.Header, maxAge time.Duration) {
 	h.Set("Cache-Control", fmt.Sprintf("max-age=%.f, immutable", maxAge.Seconds()))
 }
 
+// SetEntityTagCacheHeaders tells proxies and browsers to cache the content
+// and sets an ETag based on teleport version which can be used to check for modifications
+func SetEntityTagCacheHeaders(h http.Header, version string) {
+	h.Set("Cache-Control", "max-age=300")
```

Comment:
```suggestion
	h.Set("Cache-Control", "no-cache")
```

https://stackoverflow.com/a/55026204

Reply:
The context of that overflow post seems to be around index.html which already has a `no-cache, no-store, must-revalidate` header sent. If we do `no-cache` here, it will just be more network requests to the server. Totally fine to do it, just thought i'd try to cut back on a few.

however, after writing this, its probably more safe to always revalidate and eat a couple more network requests. Will update, thanks!